src/jrd/os/win32/winnt.cpp

```diff
@@ -863,14 +874,18 @@ static jrd_file* setup_file(Database* dbb,
 		file->fil_max_page = MAX_ULONG;
 		strcpy(file->fil_string, file_name.c_str());
 
-		if (read_only)
+		if (readOnly)
 			file->fil_flags |= FIL_readonly;
 		if (shareMode)
 			file->fil_flags |= FIL_sh_write;
+		if (forceWrites)
```

Comment:
It should be `forceWrite` without last `s`

Reply:
Checked it thrice and still something was missed :-( Sorry, to be fixed.

Reply:
Also, the MacOS build failed with:

> I/O error during "open O_CREAT" operation for file "/Users/runner/work/firebird/firebird/gen/security.tmp"
> -Error while trying to create file
> -No such file or directory
> 

Not sure if patch is the reason. thought.

Reply:
Both should be fixed now.